docs/accessing_resource_endpoints.md

```diff
@@ -25,7 +77,25 @@ k8s_resource(workload='blog-site', port_forwards='9000:4002')
 
 `localhost:9000` will be connected to container port 4002.
 
-## Naming port forwards
+## Bulk Creation of `kubectl port-forward` tunnels
+
+If you're starting out with Kubernetes and still exploring, 
+[`kubefwd`](https://kubefwd.com/) can bulk-forward all your services automatically, rather
+than forcing you to pick and choose which services you want tunnels for.
+
+Tilt's `kubefwd` operator will create port-forwards for every service
+in the namespace you're deploying to.
+
+```
+# kubefwd all namespaces Tilt deploys to.
+v1alpha1.extension_repo(name='default', url='https://github.com/tilt-dev/tilt-extensions')
+v1alpha1.extension(name='kubefwd:config', repo_name='default', repo_path='kubefwd')
+```
+
+The only downside of `kubefwd` is that Tilt will need to prompt you for `sudo`
+privileges, because it changes the DNS on your local machine.
```

Comment:
Maybe add something like "For more details, refer to the [kubefwd extension documentation](https://github.com/tilt-dev/tilt-extensions/blob/master/kubefwd/Tiltfile)" since there's no link over to that repo otherwise